TODO

```diff
@@ -0,0 +1,233 @@
+Review intervals folder: 
```

Comment:
could you please gitignore this and the changelog? It adds a lot of noise when browsing the PR from github.